content/guidelines/testing-guidelines.md

```diff
@@ -68,7 +68,58 @@ to do anything to get new functionality in the DSC modules.
 
 ### Opt-Out from Tests
 
-Not yet written.
+Default is that all the tests are opted-in. If a test cannot be resolved
+to pass then there are different scenarios supported to opt-out from tests.
+Preferably you should opt-out from tests.
+
+- Using the opt-out JSON file `.MetaTestOptOut.json`.
```

Comment:
did you not say we should just put this in Build.yaml and not even talk about that feature (so we can deprecate and remove when we feel like it?).

Reply:
Good point. Changing.